amr-wind/incflo_advance.cpp

```diff
@@ -336,7 +351,11 @@ void incflo::ApplyPredictor(bool incremental_projection)
     }
 
     // With scalars computed, compute advection of momentum
-    icns().compute_advection_term(amr_wind::FieldState::Old);
+    if (inonlin == 0) {
```

Comment:
can we do: `const auto fstate = (inonlin==0)? amr_wind::FieldState::Old: amr_wind::FieldState::NPH;` and then just `icns().compute_advection_term(fstate);`

Reply:
I changed this as such.